app/javascript/packs/articleForm.jsx

```diff
@@ -39,22 +39,7 @@ function loadForm() {
   });
 }
 
-/**
- * A function to hide an active broadcast if it exists
- * by removing a `broadcast-visible` class from it.
- *
- * @function hideActiveBroadcast
- */
-function hideActiveBroadcast() {
-  const broadcast = document.getElementById('active-broadcast');
-
-  if (broadcast) {
-    broadcast.classList.remove('broadcast-visible');
-  }
-}
-
 document.ready.then(() => {
-  hideActiveBroadcast();
```

Comment:
it was pretty easy to miss/forget that the broadcast element's class was being modified here. i also realized that we were:
1. initializing a broadcast in `initializeBroadcast`, and inserting it into the DOM with an active class
2. removing the active class here in the `articleForm` component
3. potentially re-adding a class back in the `initializeBroadcast` code [here](https://github.com/forem/forem/pull/10312/files#diff-a99b7a23c274fa6b57e161e372ad2e28L89-L90)

i don't know why i wrote the code like this originally 🤦‍♀️ but in any case, i realized that we now can simplify things a lot and just do it all directly inside of the `initializeBroadcast` 🚀